.azure-pipelines/ultimate-pipeline.yml

```
@@ -3233,7 +3229,7 @@ stages:
       - publish: system-tests/logs_remote_config_mocked_backend_live_debugging
         condition: always()
         displayName: RCM tests logs
-        artifact: _$(System.StageName)_$(Agent.JobName)_logs_rcm_$(System.JobAttempt)
+        artifact: _$(System.StageName)_$(Agent.JobName)_logs_rcm_$(System.JobAttempt)_$WEBLOG_VARIANT
```

This change shouldn't be necessary as the JobName includes the `WEBLOG_VARIANT`
```suggestion
        artifact: _$(System.StageName)_$(Agent.JobName)_logs_rcm_$(System.JobAttempt)
```